libcxx/docs/Status/SpecialMathPapers.csv

```diff
@@ -0,0 +1,2 @@
+Number,Name,Standard,Assignee,Status,First released version
+`P0226 <https://wg21.link/P0226>`_,"Mathematical Special Functions for C++17","C++17",,"|In Progress|"
```

Comment:
You don't need a list of papers if the status page is for a specific one.

Reply:
Fixed in 7d8c06d72513ed293d731915dfbb50dd001f912d.